config/enrichments/currency_conversion_config.json

```diff
@@ -11,6 +10,50 @@
 			"apiKey": "{{KEY}}",
 			"baseCurrency": "USD",
 			"rateAt": "EOD_PRIOR"
+		},
+		"mappings": [
+		{
+			"input": "iglu:ecommerce.tracking/product_entity/jsonschema/1-0-0",
+			"output": "iglu:ecommerce.tracking/converted_product_entity/jsonschema/1-0-0",
+			"fields": [
+			{
+				"inputField": "price",
+				"currency": "currency",
+				"outputField": "price"
+			},
+			{
+				"inputField": "list_price",
+				"currency": "currency",
```

Comment:
Silly question - is this the target currency that we are converting to or something else?

Reply:
It says which field from the `product_entity` schema contains the relevant currency.  Imagine the incoming context looks like this:

```
{
  "price": 42.00,
  "list_price": 42.00,
  "currency": "USD"
}
```

Reply:
Is there ever a chance that the incoming entity might contain two different currency fields?  If no, then I could move this currency setting up a level in the config.  So there is just a single currency per mapping.

Reply:
Within the same entity as part of the same event? Probably pretty unlikely as most ecomm won't support multi-currency checkouts.